library/Freckle/App/Stats.hs

```diff
@@ -267,8 +267,8 @@ sendMetric metricType name metricValue = do
 
 getEcsMetadataTags :: MonadIO m => m [(Text, Text)]
 getEcsMetadataTags = do
-  emMetadata <- runExceptT getEcsMetadata
-  either (([] <$) . err) (pure . maybe [] toTags) emMetadata
+  eMetadata <- runExceptT getEcsMetadata
+  either (([] <$) . err) (pure . toTags) eMetadata
  where
   err e = liftIO $ hPutStrLn stderr $ "Error reading ECS Metadata: " <> show e
```

Comment:
Maybe this should be softer. I worry it'll make folks think their local instance is broken since it says "Error". Perhaps, _Not configuring tags from ECS Metadata: <reason>_ is better.

Reply:
We can probably wait and see